bpf/tests/bpf_ct_tests.c

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: GPL-2.0
+/* Copyright (C) 2016-2020 Authors of Cilium */
+
+#include <bpf/ctx/skb.h>
+#include <bpf/api.h>
+
+/* most values taken from node_config.h */
```

Comment:
Probably naïve question: can't we just import `node_config.h`?

Reply:
We could, and that's what I did for an initial version but this way we get more freedom in defining only the definitions we want. 

E.g., if I remember correctly, if we include `node_config.h` we cannot directly define `CONNTRACK`, we need to define either `ENABLE_NODEPORT` or `ENABLE_HOST_FIREWALL`. I prefer just defining `CONNTRACK`.